### DISCUSSION TITLE
Application Status Still under review

### DISCUSSION THREAD

**Discussion by aaditya9899** [github/india#625] [Students]
Its been more than 3-4 days and showing my application is still under review....When will i get the confirmation pls help.

**Comment by sourabh-bhatt** [comment]
same mine application is also under review..

**Comment by Manan-collab** [comment]
I'm facing the same issue, can someone please help.

**Comment by vino905** [comment]
I am facing the same issue

**Comment by At0m0verr1de** [comment]
Same Issue

**Comment by ravikdubey360** [comment]
I am also facing the same under-process issue. please kindly resolve it.

**Comment by AllaySahoo** [comment]
i am also facing the same issue, have u guys checked the organisation section ,there it is written that we can choose max. 4 organisation,do we have to choose among them and then only our under review thing will be confirmed?

**Reply by crystal-kishore** [reply]
Noo after our profile got reviewed then only we can apply for organisations.

**Comment by AllaySahoo** [comment]
Thank you for responding. My doubt got cleared.

Thank you.


Allay Sahoo

**Comment by punitjain03** [comment]
Hi  , Currently we are reviewing the profiles, From today EOD or tomorrow morning, We will notify the status of the application status. Post that you can apply for the organisations.

**Reply by Jatinkundra** [reply]
hey @punitjain03 please verify the latest profiles also , we have to submit our proposals , please

**Reply by rungta-ayush** [reply]
Hey @punitjain03 ,
My profile is still under review and today is the deadline to submit the proposal.
I request you to please help me with it.
Thanks in advance!

**Comment by AllaySahoo** [comment]
Thanks for the update.

**Comment by s0sharma** [comment]
I applied on 29 Nov 2021, **It's been more than 72 hours** as Github as told on the page, but **my profile is still under review**. I have also mailed to github (india-externships@github.com) but didn't get any response yet. Please look into the matter and resolve it **ASAP**.

**Comment by chanchalpatil** [comment]
My application is also still under review. Awaiting response.
@punitjain03 please look into it. 
Thanks in advance.

**Comment by sourabh-bhatt** [comment]
Hello Github/India
Why yy application is nt reviewed yet?

**Comment by sourabh-bhatt** [comment]
Can anyone tell me, when my application will gonna review. It's showing my application is under review..

**Comment by skmodi649** [comment]
I don't know what is happening, my profile in under review since two weeks , can you please tell when will the review process complete, its getting too late now
